pkgs/by-name/ca/camset/package.nix

```
@@ -0,0 +1,65 @@
+{
+  python3Packages,
+  fetchFromGitHub,
+  copyDesktopItems,
+  makeDesktopItem,
+  gobject-introspection,
+  v4l-utils,
+  wrapGAppsHook3,
+  lib,
+}:
+
+python3Packages.buildPythonApplication rec {
```

This should work without `rec`.

```suggestion
python3Packages.buildPythonApplication {
```